### PR TITLE
chore: short the project name for image(#2482)

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

Co-authored-by: Ti Chi Robot <ti-community-prow-bot@tidb.io>

### What problem does this PR solve?

> Problem Summary:

It's a part of #2475;

### What's changed and how it works?

- manually cherry-pick #2482 into branch `release-2.0`. 
- do not hardcode `chaos-mesh` in the CI file, use `github.repository_owner` instead.
- add `workflow_dispatch` for manually trigger

> What's Changed:

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Chaos Dashboard`
- [ ] Need to **cheery-pick to release branches**

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Manually tested at https://github.com/STRRL/chaos-mesh/actions/runs/1433092652

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @g1eny0ung @cwen0 @YangKeao @Hexilee @iguoyr

File: .github/workflows/upload_image.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: Upload Image
 on:
+  workflow_dispatch: {}
```
Comment (g1eny0ung) [inline .github/workflows/upload_image.yml:3]: Does this mean we may need to trigger this workflow manually in the future?

Comment (STRRL) [reply .github/workflows/upload_image.yml:3]: It means we "Could", not "Need". It gives a way to build the image manually, (for debugging/profiling CI or other purposes.)

I still keep the 
```
on:
  release:
    types: [published]
```
this trigger will be triggered by tagging.

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

File: .github/workflows/upload_image.yml (hunk 1 above)
Comment (g1eny0ung) [reply .github/workflows/upload_image.yml:3]: 👌🏽Got it.

**Pull request merged**